platform/ethereum/blockbook/model_extension.go

```diff
@@ -24,6 +26,21 @@ func (t *Transaction) FromAddress() string {
 	return ""
 }
 
+func (t *Transaction) GetFee() string {
+	status, _ := t.EthereumSpecific.GetStatus()
```

Comment:
can you handle error here please

Reply:
Looks weird, but _ is not an error but status string, can be improved tho